recordit/verify-install.sh

```diff
@@ -0,0 +1,9 @@
+# test evaluates the expression as true/false
+# -d true if file exists and is a directory
+#tests if downloaded with this construct
+#tests if downloaded directly into Downloads previously
+#tests if downloaded directly into Applications previously
+
+test -d $HOME/Applications/"Recordit.app" ||
```

Comment:
Same question on testing here, if it works great, but the lack of escaped new lines gives me pause